octopoes/octopoes/models/ooi/findings.py

```diff
@@ -47,8 +47,8 @@ class FindingType(OOI):
     impact: str | None = None
     recommendation: str | None = None
 
-    risk_score: float | None = None
-    risk_severity: RiskLevelSeverity | None = None
+    risk_score: float | None = 0.0
```

Comment:
Pydantic v2 interprets this as:
- `risk_score`: a nullable field with default `0.0`, meaning we can explicitly set and initialize it to `None`, otherwise it will be assigned the default value
- `risk_severity`: a nullable field with default `RiskLevelServerity.PENDING`, meaning we can explicitly set and initialize it to `None`, otherwise it will be assigned the default value

Is this still the intended behavior? If so, leave as is. If not, for example if we don't accept `None`s anymore for the `risk_score` then we should just change the type to `float` only

Reply:
Why I did was that I'm afraid that during updating, there might be KAT installs that still have `None`s in them. Although the chance is tiny, this would break their install.

Reply:
Nice catch, indeed I do not think we need the `None`'s anymore; @noamblitz?

Reply:
I do think we need for backwards compatibility

Reply:
Addressed in cad51b0ee169995bbadb33b783017f11d8b2c06c

Reply:
As mentioned I **do not** agree with this commit.

Reply:
This will possibly brake installs.

Reply:
I misread, my bad, undoing...